test/utils/update/trust.model.fetch.spec.ts

```diff
@@ -749,7 +749,7 @@ describe("Test fetching and mapping of Trust data", () => {
       trusteeForename1: "",
       trusteeSurname: "",
       corporateIndicator: "",
-      trusteeTypeId: "5001"
+      trusteeTypeId: "5002"
```

Comment:
Former BOs now identified using trustee type 5001 instead of ceased date - so this needs to change to another trustee type.